index.html

```diff
@@ -145,7 +146,45 @@ <h4>Skills</h4>
                     <li class="inBox">Terminal</li>
                 </ul>
             </div> 
-    </section>   
+    </section>  
+    <footer>
+        <div class="space"></div>
+        <h2 class="getStar">Get started</h2>
+        <p id="headline">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+        <form class="b" 
+        action="https://formspree.io/f/xnqrlgak" method="post">
+            <div class="boxForm">
+                <label class="form-name" for="name"> 
+                    your name
+                    <input class="form-name-contend" type="text" id="name" name="user-name" maxlength="30" placeholder="enter full name" required>
+                </label>
+                <label class="form-name" for="email">
+                    your email
+                    <input class="form-name-contend" type="email" id="mail" name="user_email" placeholder="your email" required>
+                </label>
+                <label for="msg">
+                    <textarea class="form-text" id="msg" name="user_message" maxlength="500" placeholder="enter your message here..." required></textarea>
+                </label>
+                </div>
+            <div class="button">
+                <button class="blackBoton" type="submit">
+                    <a class="textBoton" href="#">Star collaboration</a>
+                </button>
```

Comment:
- To improve your user experience and also improve the structure of your page, I advise that you do not include the `a` tag in your `button` tag, this would result in your form button malfunctioning. Please correct this 🙏 🙏

Reply:
I didn't knowing that, so I change it to a paragraph, thank you

Reply:
I am not it is necessary to use a paragraph. You can just put the text in between the button tag

Reply:
ok, I will change it